G1ANT.Addon.GoogleDocs/G1ANT.Addon.GoogleDocs.Tests/Commands/googlesheet.find/ShouldFindCell_WhenSearchByValue.G1ANT

```diff
@@ -0,0 +1,12 @@
+﻿addon googledocs version 2.205.17304.1000
```

Comment:
I think you can remove "default" addons - language and core - here and in other tests.

Reply:
done